Projects/UOContent/Commands/Properties.cs

```diff
@@ -130,7 +96,7 @@ public static CPA GetCPA(PropertyInfo p)
 
                     if (CIEqual(p.Name, propertyName))
                     {
-                        var attr = GetCPA(p);
+                        var attr = Types.GetCPA(p);
```

Comment:
Is `Types` here redundant because we have a static using? 🤔

Reply:
Also I am curious how this works because `GetCPA` looks like it is still in PropertiesGump? I'll open this in an IDE later.

Reply:
Dropped `Types.` and remove dup `GetCPA` from `Types.cs` in 87903dd